web/src/SecondaryNav.tsx

```diff
@@ -71,6 +73,36 @@ class SecondaryNav extends PureComponent<NavProps> {
     let logIsSelected = this.props.resourceView === ResourceView.Log
     let alertsIsSelected = this.props.resourceView === ResourceView.Alerts
     let facetsIsSelected = this.props.resourceView === ResourceView.Facets
+    let traceIsSelected = this.props.resourceView === ResourceView.Trace
+    let spanId = this.props.span
+
+    let traceItem = null
+    if (traceIsSelected && this.props.traceUrl) {
```

Comment:
add a note here that this is to keep this tab hidden unless you know the url / `if traceIsSelected` should be removed when we make this discoverable, etc.

Reply:
done